contracts/src/base/registry/facets/distribution/v2/DelegationProxy.sol

```diff
@@ -2,20 +2,32 @@
 pragma solidity ^0.8.18;
 
 import {ERC20Votes} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+import {Initializable} from "contracts/src/diamond/facets/initializable/Initializable.sol";
 
-contract DelegationProxy {
-  address internal immutable factory;
-  address internal immutable stakeToken;
+contract DelegationProxy is Initializable {
+  address public factory;
+  address public stakeToken;
```

Comment:
consider moving this to DelegationProxyStorage lib

Reply:
Acknowledged.